app/car.py

```diff
@@ -0,0 +1,7 @@
+class Car:
```

Comment:
Where is this model used? I see that instances of this model are not created in your code.